### PR TITLE
Support reopening closed tabs

### LINKED ISSUE #1424: Reopen closed tabs
Would be nice to have an reopen feature, which brings back a recently closed search tab. On most current browsers this is achieved with pressing `CRTL + SHIFT + T`.

When pressed again, it could bring back tabs in chronical order last in first out.

### PR DESCRIPTION
**Pull request by mathiascode**
Adds Ctrl+Shift+T shortcut for reopening recently closed tabs.

Closes #1424 

I don't use this feature in other applications, so I'm not entirely sure if the behavior is correct, but feedback is welcome.

### CODE REVIEW

File: pynicotine/gtkgui/widgets/iconnotebook.py [hunk 1]
```diff
@@ -634,10 +655,12 @@ def on_pages_button_pressed(self, *args):
 
         self.popup_menu_pages.add_items(
             ("", None),
+            ("#" + _("_Reopen Closed Tab"), self.restore_removed_page),
```
Comment (slook) [inline pynicotine/gtkgui/widgets/iconnotebook.py:658]: `Reopen Closed _Tab` or `Re_open Closed Tab`

File: pynicotine/search.py [hunk 2]
```diff
@@ -132,9 +132,7 @@ def remove_all_searches(self):
     def show_search(self, token):
         events.emit("show-search", token)
 
-    def process_search_term(self, search_term, mode, room=None, user=None):
-
-        users = []
+    def process_search_term(self, search_term, mode, room=None, users=None):
```
Comment (slook) [inline pynicotine/search.py:135]: What is the reason for `user` -> `users`?

Comment (mathiascode) [reply pynicotine/search.py:135]: Was easier to deal with the internal support for multiple users this way, but on second thought, I don't know why we still support this when it's never been used. It should probably be removed.

Comment (mathiascode) [reply pynicotine/search.py:135]: Saving this for a separate PR in any case.

**Pull request merged**